tests/foreman/cli/test_environment.py

```diff
@@ -263,12 +253,6 @@ def test_positive_sc_params_by_name(self):
 
         @Assert: The command runs without raising an error
 
-        @BZ: 1219934
-
         """
         environment = make_environment()
-        Environment.sc_params({
-            'environment': environment['name'],
-        })
-        if not bz_bug_is_open(1219934):
-            self.fail('BZ #1219934 is closed, should assert the content')
+        Environment.sc_params({'environment': environment['name']})
```

Comment:
You are right, the CLI infrastructure is already checking for the return_code. There is no need to check that. Never mind about this comment.